styles/styles.css

```
@@ -1,7 +1,284 @@
 @import url("./colors.css");
 @import url("./fonts.css");
 
-.test {
-  color: var(--color-primary);
+/* ========================== HOME PAGE ================= */
+
+
+
+/* ==== MOBILE ==== */
+
+@media (max-width: 567px) {
+  .grid-container {
+    grid-template-columns: 1fr;
+  }
+
+  .poppins-style {
+    text-align: center;
+  }
+
+  .banner-style {
+    height: auto;
+    padding: 20px;
+  }
+
+  .home-page .image-container img {
+    width: 170px;
+    height: 150px;
+  }
+  
+  .home-page .text-img-container {
+    justify-content: center;
+    width: fit-content;
+    padding: 10px;
+  }
+
+  #title-banner {
+    font-size: 20px;
+    margin: 0 0 5px 0;
+  }
+
+  .icons-style {
+    gap: 10px;
+    margin: 0;
+  }
+  
+  .custom-icon {
+    width: 15px;
+    height: 15px;
+  }
+  
+  .title-products{
+    font-size: 30px;
+  }
+
+  .banner-style .poppins-style {
+    font-size: 12px;
+  }
+}
+
+/* ==== DESKTOP ==== */
+body {
+  /* ==== COMMON ==== */
+  font-family: var(--font-primary);
+  padding: 30px;
+  margin: 0;
+}
+
+/* ==== ADS (FIRST SECTION) ==== */
+.home-page .img-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.home-page .img-container img {
+  margin: 10px;
+  max-width: 100%;
+  border-radius: 20px;
+  /* ==== COMMON ==== */
+  box-shadow: 0 4px 8px var(--color-grey);
+}
+
+/* ==== GRIDS (SECOND SECTION) ==== */
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+  gap: 20px;
+  margin: 20px 0;
+}
+
+/* Title for Best Selling Products */
+
+.title-products {
+  font-weight: 600;
+  /* ==== COMMON ==== */
+  font-size: var(--title-poppins);
+}
+
+
+/* Product Box */
+.product-box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 10px;
+  /* ==== COMMON ==== */
+  background-color: var(--color-white);
+  box-shadow: 0 4px 8px var(--color-grey);
+  padding: 20px 0 10px 0;
+  position: relative;
+}
```

```suggestion
.product-box {
  display: flex;
  flex-direction: column;
  align-items: center;
  border-radius: 5px;
  background-color: var(--color-white);
  padding: 20px 0 10px 0;
  position: relative;
  border: 1px solid var(--color-grey);
  box-shadow: 1px 4px 29px -23px rgba(0, 0, 0, 0.75);
}
```

Creo que un radius menor, un box shado mas suave y  un border realmente podría hacer mas organizado el diseño. (Esta es solo una sugerencia)